### ISSUE TITLE
Unable to fetch/pull, getting 'Unable to lookup error'

### ISSUE THREAD

**Issue by Nishant-Mishra** [desktop/desktop#7591]
## Description

I am getting following error on fetch/pull (As seen on JS console)):

```js
`git -c credential.helper= pull --progress origin` exited with an unexpected code: 1.
fatal: unable to look up git.kernel.dk (port 9418) (Either the application has not called WSAStartup, or WSAStartup failed. )

error @ install.ts:23
```
My repo remote is git://git.kernel.dk/liburing.

The fetch/pull works fine with Git Command Line.

## Version

* GitHub Desktop: v1.6.6

* Operating system: MS Win 7 v6.1.7601
## Steps to Reproduce

1. Clone the mentioned Repo separately using Git.
2. SetUp Github Desktop for first use (Login and set Git credentials)
3. Add the cloned repo by drag-n-drop
4. Run 'Git Fetch' from the UI
5. Immediately an alert box appears showing mentioned error
### Expected Behavior

1. I should be asked the SSH Password 
2. Then the fetch should finish fine,
### Actual Behavior

1. See *Steps to Reproduce*

### Logs

[2019-05-21.desktop.production.log](https://github.com/desktop/desktop/files/3202156/2019-05-21.desktop.production.log)

**Comment by shiftkey** [comment]
@Nishant-Mishra thanks for the report!

This looks like you've encountered a problem with `git://` URLs and the version of Git we currently ship in Desktop. This was reported in https://github.com/git-for-windows/git/issues/1949 and fixed in Git for Windows `2.20.0`.

Can you confirm that the repository you are having a problem with was cloned using a `git://` URL?

Until we get a chance to upgrade our version of Git from 2.19.2, I believe you can workaround the issue by changing the remote URL in the repository's `.git/config` from this:

```
[remote "origin"]
        url = git://github.com/yourUsername/yourRepo.git
```

To this:

```
[remote "origin"]
        url = https://github.com/yourUsername/yourRepo.git
```

**Comment by steveward** [comment]
@Nishant-Mishra I am going to close this issue out due to inactivity. If you are still having problems with GitHub Desktop please let us know!
